backend/app/models.py

```diff
@@ -1 +1,48 @@
-# place all the model class definitions here
+from . import db
+import uuid
+from sqlalchemy.dialects.postgresql import UUID
+from datetime import datetime
+
+
+class UserMixin(object):
+    id = db.Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid4
+    )
+    name = db.Column(db.String(64), nullable=False)
+    phone_number = db.Column(db.Integer, nullable=False)
```

Comment:
I think phone number should be String and not integer, its not really a number but a collection of digits I believe

Reply:
Might overflow unless I forgot how to count

Reply:
Good point .

Reply:
APPROVE DADDY